mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -152,8 +155,17 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
             newChatHistoryManager.addChatInputMessage(input)
         }
 
-        // Step 2: Send the message to the AI
-        await _sendMessageAndSaveResponse(newChatHistoryManager)
+        // Step 2: Scroll to the bottom of the chat messages container
+        // Add a small delay to ensure the new message is rendered
+        setTimeout(() => {
+            chatMessagesRef.current?.scrollTo({
+                top: chatMessagesRef.current.scrollHeight,
+                behavior: 'smooth'
+            });
+        }, 100);
+
+        // Step 3: Send the message to the AI
+        await _sendMessageAndSaveResponse(newChatHistoryManager)     
```

Comment:
The smooth scrolling is awesome! Feels super polished. 

The one change I think we could make is that we scroll when the user submits the message, but then we don't scroll again when we get a response. The result for me is that I can usually only see the first line of the AI generated message. Like this: 

<img width="367" alt="Screenshot 2024-12-19 at 6 14 15 PM" src="https://github.com/user-attachments/assets/9d006be9-6b74-45f4-a1cd-d5be5e0f6a71" />

I think a nice fix could be that once the Ai generated response is created, we scroll so that the taskpane is just the most recent user input message and Ai response. 

Just make sure to handle the case where the user's input message is really long like this:

```python
Deduplicate this dictionary's keys

finance_data = {
    'Date': ['2024-01-01', '2024-01-02', '2024-01-03', '2024-01-04', '2024-01-05'],
    'Revenue': [1000, 1500, 2000, 2500, 3000],
    'Expenses': [500, 700, 800, 900, 1200],
    'Net_Profit': [500, 800, 1200, 1600, 1800],
    'Customer_Count': [50, 75, 100, 125, 150],
    'Marketing_Expenses': [200, 250, 300, 350, 400],
    'R&D_Expenses': [150, 200, 250, 300, 350],
    'Sales_Expenses': [100, 150, 200, 250, 300],
    'Operating_Income': [300, 550, 900, 1250, 1400],
    'Gross_Profit': [500, 800, 1200, 1600, 1800],
    'Total_Assets': [5000, 5200, 5400, 5600, 5800],
    'Total_Liabilities': [2000, 2100, 2200, 2300, 2400],
    'Equity': [3000, 3100, 3200, 3300, 3400],
    'Debt_to_Equity_Ratio': [0.67, 0.68, 0.69, 0.70, 0.71],
    'Current_Ratio': [1.5, 1.6, 1.7, 1.8, 1.9],
    'Quick_Ratio': [1.2, 1.3, 1.4, 1.5, 1.6],
    'Return_on_Assets': [0.1, 0.12, 0.15, 0.18, 0.2],
    'Return_on_Equity': [0.15, 0.16, 0.18, 0.2, 0.22],
    'Earnings_per_Share': [1.0, 1.2, 1.5, 1.8, 2.0],
    'Price_to_Earnings_Ratio': [15, 16, 17, 18, 19],
    'Dividend_Yield': [0.02, 0.025, 0.03, 0.035, 0.04],
    'Market_Cap': [10000, 10500, 11000, 11500, 12000],
    'Cash_Flow': [300, 400, 500, 600, 700],
    'Inventory_Turnover': [5, 6, 7, 8, 9],
    'Days_Sales_Outstanding': [30, 28, 26, 24, 22],
    'Total_Assets': [5000, 5200, 5400, 5600, 5800],
    'Total_Liabilities': [2000, 2100, 2200, 2300, 2400],
    'Equity': [3000, 3100, 3200, 3300, 3400],
    'Debt_to_Equity_Ratio': [0.67, 0.68, 0.69, 0.70, 0.71],
    'Current_Ratio': [1.5, 1.6, 1.7, 1.8, 1.9],
    'Quick_Ratio': [1.2, 1.3, 1.4, 1.5, 1.6],
    'Return_on_Assets': [0.1, 0.12, 0.15, 0.18, 0.2],
    'Return_on_Equity': [0.15, 0.16, 0.18, 0.2, 0.22],
    'Earnings_per_Share': [1.0, 1.2, 1.5, 1.8, 2.0],
    'Price_to_Earnings_Ratio': [15, 16, 17, 18, 19],
    'Dividend_Yield': [0.02, 0.025, 0.03, 0.035, 0.04],
    'Market_Cap': [10000, 10500, 11000, 11500, 12000],
    'Cash_Flow': [300, 400, 500, 600, 700],
    'Inventory_Turnover': [5, 6, 7, 8, 9],
    'Days_Sales_Outstanding': [30, 28, 26, 24, 22]
}
```